packages/providers/lib.esm/base-provider.d.ts

```diff
@@ -139,7 +128,6 @@ export declare class BaseProvider extends Provider implements EnsProvider {
     getTransactionReceipt(transactionHash: string | Promise<string>): Promise<TransactionReceipt>;
     getLogs(filter: Filter | FilterByBlockHash | Promise<Filter | FilterByBlockHash>): Promise<Array<Log>>;
     getEtherPrice(): Promise<number>;
-    _getBlockTag(blockTag: BlockTag | Promise<BlockTag>): Promise<BlockTag>;
     getResolver(name: string): Promise<null | Resolver>;
```

Comment:
I think that `getResolver`, `_getResolver`, `resolveName`, `lookupAddress` and `getAvatar` are part of ENS and we should log another issue in ZenHub with lower priority to think about the ENS support